CI/travis/ci-linux.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -x
+
+handle_centos() {
+	local package=$1
+
+	yum -y install yum-utils gcc
+	yum config-manager --set-enabled powertools
+
+	pwd
+
+	cd /opt
+	wget https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+	tar xzf Python-3.5.1.tgz
+	cd Python-3.5.1
+	./configure
+	make altinstall
+	cd /ci
+
+	ls -al /usr/local/lib
+
+	yum -y install epel-release
+
+	yum -y install bzip2 gzip
+
+	yum localinstall -y $package
+
+	python3.5 -m pip install pylibiio --no-binary :all:
+
+	# Build project
+	sudo ln -fs /usr/bin/python3.5 /usr/bin/python
+	python3.5 -m ensurepip
+	mkdir -p build
+	cd build
+	cmake -DPYTHON_BINDINGS=ON -DENABLE_PACKAGING=ON ..
+	sudo make && sudo make package && make test
+	sudo make install
+	ldconfig
+	cd ..
+	cd bindings/python
+	pip install -r requirements_dev.txt
```

Comment:
There are mixed "_pip install <package_name>_" with "_python3.5 -m pip install <package_name>_". In most of the cases they should do the same thing, but also may generate errors whenever there are multiple versions of python, or PATH variable is not set properly (you can see some answers here: [https://stackoverflow.com/questions/40392499/why-is-m-needed-for-python-m-pip-install](https://stackoverflow.com/questions/40392499/why-is-m-needed-for-python-m-pip-install))